framework/src/setroubleshoot/server.py

```diff
@@ -115,6 +115,13 @@ def sighandler(signum, frame):
         return
 
 
+def polling_failled_handler(signum, frame):
```

Comment:
there is a typo, it should probably be 'polling_failed_handler'

Reply:
Thanks.